packages/theme/src/cli/commands/theme/update/init.ts

```diff
@@ -0,0 +1,18 @@
+import ThemeCommand from '../../../utilities/theme-command.js'
+import {init} from '../../../services/update/init.js'
+import {themeFlags} from '../../../flags.js'
+import {globalFlags} from '@shopify/cli-kit/node/cli'
+
+export default class UpdateInit extends ThemeCommand {
+  static description = `Initialize an 'update_extension.json' script.`
+
+  static flags = {
+    ...globalFlags,
+    path: themeFlags.path,
+  }
+
+  async run(): Promise<void> {
+    const {flags} = await this.parse(UpdateInit)
+    await init(flags.path)
```

Comment:
Should we return the newly created file path as a result of `init`? I noticed we don't. I assumed it's be in the root but i wasn't sure until i checked. What's the best practice?

Reply:
Yeah, we're actually returning the relative path of the file, as we may notice here:

![Screenshot 2023-05-25 at 12 18 10](https://github.com/Shopify/cli/assets/1079279/38f81642-c683-4b1d-bffb-ef22c099d639)

Initially, we were showing the absolute path, but after taking another look at the context, we realized that using the relative path is actually the best practice in this situation. You can read more about it in this [here](https://github.com/Shopify/cli/pull/1909#discussion_r1195737066) :)